public/scripts/customScripts.js

```diff
@@ -8,7 +8,7 @@ const defaultSessionData = JSON.stringify({
   fields: {},
   sort: 'best_match',
   filters: { resourceType: 'all' },
-  rowsPerPage: 20,
+  rowsPerPage: '20',
```

Comment:
are you sure it needs to  be a string?

Reply:
The dropdown option should be selected based on the string value, so I have changed to string. With numeric, this won't cause any problems since the default option is always selected when nothing is selected. The default option is also 20, so this won't really cause any problems. So it's better to put it as a string, as this won't cause any issues with the elasticsearch query. But to make sure, we are now converting this value to integer while requesting the search results. Now the elasticsearch querybuilder receives the integer.